src/Database/Statement/StatementDecorator.php

```diff
@@ -64,7 +64,7 @@ class StatementDecorator implements StatementInterface, Countable, IteratorAggre
      *  such as PDOStatement.
      * @param \Cake\Database\Driver|null $driver Driver instance
      */
-    public function __construct($statement = null, $driver = null)
+    public function __construct($statement, $driver = null)
```

Comment:
The doc block states |null still.

Reply:
Ah you want it to allow null, but the argument has to be present.

Reply:
Actually i don't want to change anything. I was just trying something and forgot to revert :stuck_out_tongue:.